debezium-embedded/src/test/java/io/debezium/embedded/CompletionTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.embedded;
+
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.junit.Test;
+
+/**
+ * A unit test that helps identify the functionality and behavior of {@link CompletableFuture}.
+ * @author Randall Hauch
+ */
+public class CompletionTest {
```

Comment:
do we requires the tests from this class ? It seems from the total duration of `Thread.sleep` calls to add some overhead to the DBZ build.